app/src/main/res/values/strings.xml

```diff
@@ -1,17 +1,15 @@
 <resources>
     <string name="app_name">Raise To Answer</string>
-    <string name="permissions_needed">App can\'t work without these permission, sorry…</string>
+    <string name="permissions_needed">Grant the permissions to use the app.</string>
     <string name="raise_to_answer_header">Raise To Answer</string>
     <string name="incoming_call_service">Incoming call service</string>
-    <string name="raise_to_answer_is_listening_to_sensor_data">Raise To Answer is listening to sensor data</string>
+    <string name="raise_to_answer_is_listening_to_sensor_data">Listening to sensor data…</string>
     <string name="privacy_policy">Privacy Policy</string>
-    <string name="could_not_bind_sensor">Failed registering sensors, your device may lack a proximity sensor or accelerometer. Raise To Answer cannot continue.</string>
-    <string name="feature_raise_to_answer">Answer by holding your phone to your ear</string>
-    <string name="feature_flip_over_to_decline">Decline by putting your phone down screen-first</string>
-    <string name="detected_raise_to_answer">Call pickup detected</string>
-    <string name="detected_flip_over">Call decline detected</string>
+    <string name="could_not_bind_sensor">Could not find any proximity sensor or accelerometer on your device. Cannot continue.</string>
+    <string name="feature_raise_to_answer">Answer by holding your device up to your ear</string>
+    <string name="feature_flip_over_to_decline">Hang up without answering by putting your device down screen-first</string>
```

Comment:
Not a fan of renaming "decline" to "hang up without answering". It's way longer and "decline" is a really well-known word that every phone user really will know.

Reply:
It breaks down into "reject. vs. decline", with some other options.
There is nothing for hold/postpone, so there is no consistency in using all the technical terms that way. The other strings already hinge on "hang up".

"Call decline detected" is not idomatic.